_includes/header.html

```diff
@@ -11,11 +11,11 @@
       <ul class="nav navbar-nav mainbar-nav">
 
       <li>
-      <a href="/"><img src="http://parliamentwatch.ug/wp-content/uploads/2016/06/logo-5.png" width="80%" height="50%" alt="Parliament Watch Uganda"></a>
+      <a href="/uganda-parliament-watch"><img src="http://parliamentwatch.ug/wp-content/uploads/2016/06/logo-5.png" width="80%" height="50%" alt="Parliament Watch Uganda"></a>
```

Comment:
This will need to be `<a href="{{ "/" | prepend: site.baseurl }}">...` so that it works when you're running locally and on the live site. See [this blog post](https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/) for more details about `baseurl` in Jekyll.